UpdateUser/handler.ts

```diff
@@ -4,7 +4,8 @@ import { User } from "@azure/graph/esm/models";
 import * as express from "express";
 import { toError } from "fp-ts/lib/Either";
 import { identity } from "fp-ts/lib/function";
-import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import { fromLeft } from "fp-ts/lib/TaskEither";
+import { taskEither, TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
```

Comment:
If you accept [that](https://github.com/pagopa/io-functions-admin/pull/92/files/fcaa6c49c6d6c31c93e5ab9bed0fbdef5e809c04..924224cb476947ebace8390e87d0425b7b69c5ad#r504752786) suggestion you'll need this, too

```suggestion
import { fromEither, taskEither, TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
```

Reply:
My bad, you' re right 😄